src/converters/lintConfigs/rules/convertRules.test.ts

```diff
@@ -353,6 +353,45 @@ describe("convertRules", () => {
         ]).toContainEqual(converted);
     });
 
+    it("merges when a rule's ruleSeverity explicitly differs from its TSLint equivalent", () => {
+        // Arrange
+        const conversionResult: {
+            rules: { ruleName: string; ruleSeverity: ESLintRuleSeverity }[];
+        } = {
+            rules: [
+                {
+                    ruleName: "eslint-rule-a",
+                    ruleSeverity: "off",
+                },
+            ],
+        };
+        const { tslintRule, converters, mergers } = setupConversionEnvironment({
+            ruleSeverity: "error",
+            conversionResult,
+            ruleToMerge: conversionResult.rules[0].ruleName,
+        });
+
+        // Act
+        const { converted } = convertRules(
+            { ruleConverters: converters, ruleMergers: mergers },
+            { [tslintRule.ruleName]: tslintRule },
+            new Map<string, string[]>(),
```

Comment:
Nit: the type arguments can be inferred here.

```suggestion
            new Map(),
```

Eventually we'll have a `no-inferrable-type-arguments` rule or some such in typescript-eslint... one day!